assets/css/styles.css

```diff
@@ -0,0 +1,29 @@
+:root {
```

Comment:
Not sure if this is just a placeholder for now or the finalised colour scheme for the project. Please discuss with your team to confirm the colours.